hygen/_templates/promptUtils.js

```diff
@@ -37,20 +37,25 @@ async function getFiles(dir) {
     }, []);
 }
 
-async function doesFileExists (pathArray) {
```

Comment:
@albertoarana how about keeping the **async** (and removing the await)? I know it's not required, but it's more clear to the client using it, that the function is async

Reply:
I agree :)